### PR TITLE
move npm-run-all to devDependencies

### PR DESCRIPTION
**Pull request by gr2m**
follow up for https://github.com/lquixada/cross-fetch/commit/9af3c4447192933f00ee52b705ffb6edcb1dfd3d#r31435921

### CODE REVIEW

**Comment by lquixada** [comment]
hi! thanks for the PR! can you squash the commit and follow the message pattern from the git history? it should look like this: "moved npm-run-all to devDependencies."

File: package.json [hunk 1]
```diff
@@ -73,6 +72,7 @@
     "lint-staged": "8.1.0",
     "mocha": "5.2.0",
     "mocha-headless-chrome": "2.0.1",
+    "npm-run-all": "^4.1.3",
```
Comment (lquixada) [inline package.json]: the ^ is not needed!

Comment (gr2m) [reply package.json]: no problem, will fix! Out of curiosity, why do you pin the version numbers? Especially the production version numbers, that can cause that multiple versions of the same dependency are necessary, as the pinned version prevents deduping of packages

**Comment by gr2m** [comment]
I’ve seen you have both a package-lock.json and a yarn.lock file? I only updated package-lock.json now, I think you should have one or the other?

**Comment by lquixada** [comment]
package-lock.json shouldn't exist! it was added by accident.

File: package.json (hunk 1 above)
Comment (lquixada) [reply package.json]: Been explained here: #13

**Comment by gr2m** [comment]
can you finish this up? I don’t use yarn myself, I'd need to install it only to create the lock file :)

**Comment by lquixada** [comment]
hmmm... if you don't use yarn, i think it's easier to do the fix myself if you don't mind. Otherwise once this PR is merged, I'll have to create another commit only to update the yarn.lock with the change you made in the package.json. This is supposed to be a simple change.

**Comment by gr2m** [comment]
> it's easier to do the fix myself if you don't mind

Not at all! You can also edit my PR directly, I do it all the time in my projects

**Comment by lquixada** [comment]
cool! I'll do that in the terminal, it'll be quicker! Thanks for reporting the issue and the PR!

**Pull request closed without merging**